README.md

```diff
@@ -25,9 +25,9 @@ The number of runners on a single unit is configured using two configuration opt
 For example, if the charm is deployed with 2 units `juju deploy <charm> -n 2` and the `containers` value of 3 is in use,
 there will be a total of 6 container based runners, three on each unit.
 
-## Reconcilation
+## Reconciliation
 
-Each unit will periodically check the number of idle runners at an interval that is configured using the `check-interval` setting to maintain the configured number of runners. During the check, all offline runners are unregistered from GitHub and corresponding containers or virtual machines are destroyed. 
+Each unit will periodically check the number of idle runners at an interval that is configured using the `check-interval` setting to maintain the configured number of runners. During the check, all offline runners are unregistered from GitHub and corresponding containers or virtual machines are destroyed.
```

Comment:
"Each unit will periodically check the number of idle runners at an interval that is configured using the `check-interval` setting to maintain the configured number of runners." -> "Each unit will periodically check the number of idle runners at the interval specified by `check-interval` to maintain the appropriate number."

Reply:
"During the check, all offline" -> "During the check, all the offline"

Reply:
fixed